packages/web/src/builders/package/package.impl.ts

```diff
@@ -166,21 +168,24 @@ export function run(
       } else {
         context.logger.info('Bundling...');
         return from(rollupOptions).pipe(
-          mergeScan(
-            (acc, options) =>
-              runRollup(options).pipe(
-                map(result => {
-                  return {
-                    success: acc.success && result.success
-                  };
-                })
-              ),
+          mergeMap(options => runRollup(options)),
+          scan(
```

Comment:
I think the `scan` here is actually a mistake now. What we should do it bail on the first error... so the `catchError` in the other file should actually be moved here.

Basically the behavior should be:
1. Builds are run in parallel (i.e. `mergeMap`)
2. The first error should bail the entire build.
3. Any build errors (e.g. type errors) should only be printed once.

Reply:
very good catch regarding the multiple errors! I made the changes and tried them out - it prints only once now

though it still prints once per successful bundle, as it did before, which I think is okay (or at least not worth spending too much consolidating into a single box):

![image](https://user-images.githubusercontent.com/5975076/75149132-2ec92100-56f9-11ea-97fa-da5ef444a89f.png)

Reply:
The successful outputs are useful I think anyway since they provide the location to where the bundles are and some metadata.